tests/test_pairing_handler.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Missing copyright info, but this can be tackled on another PR

Reply:
Yes, it could be appropriate to address the problem on all test files at once in a separate PR.